src/components/molecules/ChatPoll/ChatPoll.tsx

```diff
@@ -120,6 +123,18 @@ export const ChatPoll: React.FC<ChatPollProps> = ({
     ));
   }, [questions, calculateVotePercentage]);
 
+  const renderPollContent = useMemo(() => {
+    if (isVoting) {
+      return <Loading />;
+    }
+
+    if (hasVoted || isMine) {
+      return renderResults;
+    }
+
+    return renderQuestions;
+  }, [hasVoted, isVoting, isMine, renderQuestions, renderResults]);
```

Comment:
Not sure that it makes sense to `useMemo` this. It's not really doing anything 'heavy' within it to make it worth memoing.

Reply:
got it! fixed in 3e4c5f3310eeb05fddf062dec5155638f3125926